RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Stream Engine
 
+# Next Release
```

Comment:
Specify this one as In-Development 1.3.9 2017-10-18

Reply:
Done